--- /home/vk352/anaconda3/envs/rnns_vk/lib/python3.6/site-packages/torch/nn/modules/activation.py
+++ /home/vk352/anaconda3/envs/rnns_vk/lib/python3.6/site-packages/torch/nn/modules/activation.py
@@ -1,16 +1,15 @@
 class Tanh(Module):
-    r"""Applies the Hyperbolic Tangent (Tanh) function element-wise.
-
-    Tanh is defined as:
+    r"""Applies the element-wise function:
 
     .. math::
-        \text{Tanh}(x) = \tanh(x) = \frac{\exp(x) - \exp(-x)} {\exp(x) + \exp(-x)}
+        \text{Tanh}(x) = \tanh(x) = \frac{e^x - e^{-x}} {e^x + e^{-x}}
 
     Shape:
-        - Input: :math:`(*)`, where :math:`*` means any number of dimensions.
-        - Output: :math:`(*)`, same shape as the input.
+        - Input: :math:`(N, *)` where `*` means, any number of additional
+          dimensions
+        - Output: :math:`(N, *)`, same shape as the input
 
-    .. image:: ../scripts/activation_images/Tanh.png
+    .. image:: scripts/activation_images/Tanh.png
 
     Examples::
 
@@ -19,6 +18,7 @@
         >>> output = m(input)
     """
 
-    def forward(self, input: Tensor) -> Tensor:
+    @weak_script_method
+    def forward(self, input):
         return torch.tanh(input)
 